app/src/main/java/com/daniel/card_game_android/fragments/FragmentMap.java

```diff
@@ -0,0 +1,96 @@
+package com.daniel.card_game_android.fragments;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.fragment.app.Fragment;
+
+import com.daniel.card_game_android.R;
+import com.daniel.card_game_android.objects.Record;
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.MapView;
+import com.google.android.gms.maps.MapsInitializer;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.model.CameraPosition;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MarkerOptions;
+
+public class FragmentMap extends Fragment {
+
+    MapView mMapView;
+    double latitude = 51.501024, longitude = -0.142666;
+    String name = "Buckingham Palace", score = "Always my winner";
```

Comment:
why?

Reply:
Only a a default one. We can change it for the first in the list...